### PR TITLE
mrs(3): improve text about AT_BSDFLAGS

### PR DESCRIPTION
**Pull request by brooksdavis**
Hopefully improve the previous mangled sentence.

### CODE REVIEW

**Comment by bsdjhb** [comment]
So I was going to try to take a stab at rewording the paragraph in question a bit further, but upon reflection I realized that the `_FORCE` flag isn't used by MRS at all anymore after removing the `malloc_revocation` weak symbol.  Maybe we should just remove the `_FORCE` flag entirely which simplifies the language here as one can then just say that the `CHERI_REVOKE` flag from `AT_BSDFLAGS` is used to enable revocation at runtime.

**Comment by brooksdavis** [comment]
I've taken another shot at it.

**Review by bsdjhb — APPROVED** [review]
I do wonder if we want to also remove the _FORCED flag entirely from the implementation as well?

**Comment by brooksdavis** [comment]
I think we could remove it relatively safely.

File: sys/kern/imgact_elf.c [hunk 1]
```diff
@@ -1847,16 +1847,13 @@ __elfN(freebsd_copyout_auxargs)(struct image_params *imgp, uintcap_t base)
 	 * Precedence: procctl, ELF note, system default.
 	 * In case of conflicting flags, disable wins.
 	 */
-	if ((imgp->proc->p_flag2 & P2_CHERI_REVOKE_MASK) != 0) {
-		bsdflags |= ELF_BSDF_CHERI_REVOKE_FORCED;
-		if ((imgp->proc->p_flag2 & P2_CHERI_REVOKE_DISABLE) == 0)
-			bsdflags |= ELF_BSDF_CHERI_REVOKE;
+	if ((imgp->proc->p_flag2 & P2_CHERI_REVOKE_MASK) != 0 &&
```
[the remark this answers is not in the document]
Comment (brooksdavis) [reply sys/kern/imgact_elf.c:1850]: Indeed, will fix.

**Review by bsdjhb — APPROVED** [review]

**Pull request merged**